bigtable/bttest/inmem.go

```diff
@@ -165,13 +165,6 @@ func (s *Server) Close() {
 	s.srv.Stop()
 	_ = s.l.Close()
 
-	var tbls []*table
```

Comment:
@dragonsinth staticcheck found this problem. Should we skip it?

Reply:
Ah, good catch but this is the wrong fix.  Rather than deleting this block, it should be:
```
	for _, tbl := range tbls {
```
below